lang/python/chardet/Makefile

```diff
@@ -15,38 +15,50 @@ PKG_LICENSE:=LGPL-2.1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://pypi.python.org/packages/fc/bb/a5768c230f9ddb03acc9ef3f0d4a3cf93462473795d18e9535498c8f929d/
 PKG_HASH:=84ab92ed1c4d4f16916e05906b6b75a6c0fb5db821cc65e70cbd64a3e2a5eaae
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-chardet-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/chardet
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/chardet/Defaults
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
   MAINTAINER:=Gergely Kiss <mail.gery@gmail.com>
-  TITLE:=Universal encoding detector for Python 2 and 3
   URL:=https://github.com/chardet/chardet
-  DEPENDS:=+python
 endef
 
-define Package/chardet/description
-  Universal encoding detector for Python 2 and 3
+define Package/chardet
+$(call Package/chardet/Default)
+  TITLE:=Universal encoding detector for Python 2
+  DEPENDS:=+PACKAGE_chardet:python
```

Comment:
similar, let's rename to `python-chardet`

Reply:
Ok